doc/api/typescript.md

```diff
@@ -82,20 +93,24 @@ The `tsconfig.json` option `allowImportingTsExtensions` will allow the
 TypeScript compiler `tsc` to type-check files with `import` specifiers that
 include the `.ts` extension.
 
-### Unsupported TypeScript features
+### TypeScript features
 
 Since Node.js is only removing inline types, any TypeScript features that
-involve _replacing_ TypeScript syntax with new JavaScript syntax will error.
-This is by design. To run TypeScript with such features, see
-[Full TypeScript support][].
+involve _replacing_ TypeScript syntax with new JavaScript syntax will error,
+unless the flag [`--experimental-type-transform`][] is passed.
 
-The most prominent unsupported features that require transformation are:
+The most prominent features that require transformation are:
 
 * `Enum`
-* `experimentalDecorators`
 * `namespaces`
+* `legacy module`
 * parameter properties
 
+Since Decorators are currently a [TC39 ECMA402 stage 3 proposal](https://github.com/tc39/proposal-decorators)
```

Comment:
FWIW, ECMA-402 is the internationalization API, e.g. `globalThis.Intl`. [ECMA-262](https://tc39.es/ecma262/) is the specification for ECMAScript. Anyway, mentioning "TC39 Stage 3 proposal" or "ECMAScript Stage 3 proposal" should be sufficient.

```suggestion
Since Decorators are currently a [TC39 stage 3 proposal](https://github.com/tc39/proposal-decorators)
```